native/swift/Tests/wordpress-api/LoginTests.swift

```diff
@@ -42,15 +43,21 @@ class LoginTests {
 
         let client = WordPressLoginClient(requestExecutor: stubs)
 
-        await #expect(performing: {
+        await #expect(
+performing: {
             _ = try await client.login(
                 site: "https://example.com/blog",
                 appName: "foo",
                 appId: appId,
-                authenticator: Authenticator()
-            ).get()
-        }, throws: { error in
-            guard let loginError = error as? WordPressLoginClient.Error, case .invalidSiteAddress = loginError else {
+                authenticator: MockAuthenticator
+                    .returning(
+                        .failure(WordPressLoginClient.Error.invalidSiteAddress(.UrlDiscoveryFailed(attempts: .init())))
+                    )
+            )
+        },
+ throws: { error in
+            guard let loginError = error as? WordPressLoginClient.Error,
+ case .invalidSiteAddress = loginError else {
```

Comment:
Fix indentations (performing, throws, case) here?

Reply:
Should be sorted!